docs/manpages/zone2ldap.1.md

```diff
@@ -20,6 +22,15 @@ inserted to an LDAP server.
 --basedn=*DN*
 :    Base DN to store objects below
 
+--create-ent
```

Comment:
Just to sum up what we discussed on IRC: I'll remove this option and let zone2ldap do The Right Thing©® from the start.